src/client/containers/OrderConfirmationPage/OrderConfirmationPage.Container.js

```diff
@@ -2,5 +2,30 @@ import React from 'react';
 import './OrderConfirmationPage.styles.css';
 
 export default function OrderConfirmationPage() {
-  return <div>Order Confirmation Page</div>;
+  return <div className="confirmation-page">
```

Comment:
Nitpick(readability):
The formatting is a little off in this file, this makes it hard to tell which content belongs in which box.